UnattendResources/Logon.ps1

```diff
@@ -74,47 +118,103 @@ function Release-IP {
         }
 }
 
-try
-{
-    Import-Module "$resourcesDir\ini.psm1"
-    $installUpdates = Get-IniFileValue -Path $configIniPath -Section "DEFAULT" -Key "InstallUpdates" -Default $false -AsBoolean
-    $persistDrivers = Get-IniFileValue -Path $configIniPath -Section "DEFAULT" -Key "PersistDriverInstall" -Default $true -AsBoolean
+function Install-WindowsUpdates {
+    Import-Module "$resourcesDir\WindowsUpdates\WindowsUpdates"
+    $BaseOSKernelVersion = [System.Environment]::OSVersion.Version
+    $OSKernelVersion = ($BaseOSKernelVersion.Major.ToString() + "." + $BaseOSKernelVersion.Minor.ToString())
 
-    if($installUpdates)
+    #Note (cgalan): We are adding some updates to a blacklist because

```

Comment:
Some updates are black-listed as they are either failing to install or superseeded by the newer updates.